extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ModeledConnection.java

```diff
@@ -117,12 +117,18 @@
     public static final String MAX_CONNECTIONS_PER_USER_NAME = "max-connections-per-user";
 
     /**
+     * The connection weight for weighted algorithms.
+     */
+    public static final String CONNECTION_WEIGHT = "weight";
+
+    /**
      * All attributes related to restricting user accounts, within a logical
      * form.
      */
     public static final Form CONCURRENCY_LIMITS = new Form("concurrency", Arrays.<Field>asList(
         new NumericField(MAX_CONNECTIONS_NAME),
-        new NumericField(MAX_CONNECTIONS_PER_USER_NAME)
+        new NumericField(MAX_CONNECTIONS_PER_USER_NAME),
+        new NumericField(CONNECTION_WEIGHT)
```

Comment:
That's fine with me - I was trying to avoid creating too many section breaks, especially for sections with a single item in them, and it seemed like this still fell under the heading of "Concurrency Limits", since it is used to impact how connections are concurrently applied to systems, but I'll break it out into another section.